blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_3b.py

```diff
@@ -0,0 +1,66 @@
+"""Formula 5.3b from NEN-EN 1992-1-1+C2:2011: Chapter 5 - Structural Analysis."""
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_negative
+
+
+class Form5Dot3BTransverseForceBracedMembers(Formula):
+    """Class representing formula 5.3b for the calculation of the transverse force for braced members, :math:`H_{i}`.
+
+    See Figure 5.1 a2.
+    """
+
+    label = "5.3b"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        theta_i: DIMENSIONLESS,
+        n_axial_force: KN,
+    ) -> None:
+        """[:math:`H_{i}`] Transverse force for unbraced members [:math:`kN`].
```

Comment:
It is actually for braced members
![afbeelding](https://github.com/Blueprints-org/blueprints/assets/44614728/4a31c263-c23a-4bbc-b07b-2fd94a43f39a)